hw/xbox/nv2a/nv2a_regs.h

```diff
@@ -674,7 +674,8 @@
 #   define NV_PRAMDAC_PLL_TEST_COUNTER_NVPLL_LOCK              (1 << 29)
 #   define NV_PRAMDAC_PLL_TEST_COUNTER_MPLL_LOCK               (1 << 30)
 #   define NV_PRAMDAC_PLL_TEST_COUNTER_VPLL_LOCK               (1 << 31)
-
+#define NV_PRAMDAC_GENERAL_CONTROL                       0x00000600
+#   define NV_PRAMDAC_GENERAL_CONTROL_ALT_MODE                 (1 << 12)
```

Comment:
Use `NV_PRAMDAC_GENERAL_CONTROL_ALT_MODE_SEL` to describe bit value, not just the field